GradedModulePresentationsForCAP/gap/GradedModulePresentations.gi

```diff
@@ -604,7 +614,17 @@ InstallGlobalFunction( ADD_GRADED_COLIFT,
         colift := Colift( UnderlyingPresentationMorphism( alpha ), UnderlyingPresentationMorphism( beta ) );
         
         if colift = fail then
-            return fail;
+        
+          return fail;
+        
+        fi;
+        
+        colift := GradedPresentationMorphism( Range( alpha ), colift, Range( beta ) );
```

Comment:
You forgot to clean the colift function from its temporary lines.

Reply:
Thx, removed it.